src/pages/VeSDL/index.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  Alert,
+  Box,
+  Button,
+  Container,
+  Divider,
+  Link,
+  Paper,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material"
+import React, { useState } from "react"
+import ArrowDownwardIcon from "@mui/icons-material/ArrowDownward"
+import { DatePicker } from "@mui/lab"
+import GaugeVote from "./GaugeVote"
+import LockedInfo from "./LockedInfo"
+import TokenInput from "../../components/TokenInput"
+import { useTranslation } from "react-i18next"
+
+export default function VeSDL(): JSX.Element {
+  const [date, setDate] = useState<string | null>(null)
+  const [sdlTokenVal, setSdlTokenVal] = useState<string>("0.0")
+  const { t } = useTranslation()
+  const handleChange = (value: string) => {
+    setSdlTokenVal(value)
+  }
+  return (
+    <Container sx={{ py: 3 }}>
+      <Box display={{ sm: "flex" }} gap={2}>
+        <Stack flex={1} spacing={2} mb={2}>
+          <Paper
+            sx={{
+              display: "flex",
+              flexDirection: "column",
+              gap: 2,
+              p: 4,
+            }}
+          >
+            <Typography variant="h2" textAlign="center">
+              {t("sdlLock")}
+            </Typography>
+
+            <TokenInput
+              data-testid="sdlTokenInput"
+              allowDecimalOverflow
+              symbol="SDL"
+              name="sdl"
+              max="3000"
+              onChange={handleChange}
+              inputValue={sdlTokenVal}
+            />
+            <Box display="flex" alignItems="center">
+              <div>
+                <Typography mr={1} noWrap>
+                  Unlock date:
+                </Typography>
+              </div>
+              <DatePicker
```

Comment:
lets add `minDate` equal to one day in the future.

Reply:
Fortunately, `DatePicker `is having `minDate` by default, I think we can use this without any modify.